sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-bitmap.js

```diff
@@ -89,7 +89,7 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         gl.texParameteri(bindingTarget, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
         // Set up pixel store parameters
         gl.pixelStorei(gl.UNPACK_FLIP_Y_WEBGL, flipY);
-        gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, premultiplyAlpha);
+        gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, false);
```

Comment:
After taking a closer look, I think there is another bug somewhere and I don't think it would be a quick fix. Is it possible to merge the "temporary remove the imagebitmap tests" only? Or should I create a separate pull request? I will re-enable these tests once the bug is fixed. Thank you.